### PR TITLE
Net examples

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @steveklabnik

### CODE REVIEW

File: src/libstd/net/ip.rs [hunk 1]
```diff
@@ -176,8 +193,18 @@ impl Ipv4Addr {
     /// Returns true for the special 'unspecified' address (0.0.0.0).
     ///
     /// This property is defined in _UNIX Network Programming, Second Edition_,
-    /// W. Richard Stevens, p. 891; see also [ip7]
-    /// [ip7][http://man7.org/linux/man-pages/man7/ip.7.html]
+    /// W. Richard Stevens, p. 891; see also [ip7].
+    ///
+    /// [ip7]: (http://man7.org/linux/man-pages/man7/ip.7.html)
```
Comment (frewsxcv) [inline src/libstd/net/ip.rs]: Should this URL be in parenthesis?

Comment (GuillaumeGomez) [inline src/libstd/net/ip.rs]: No, it shouldn't.

**Pull request merged**